src/site/markdown/authentication.md

```diff
@@ -0,0 +1,82 @@
+---
+title: SCM Authentication
+---
+
+<!-- Licensed to the Apache Software Foundation (ASF) under one-->
+<!-- or more contributor license agreements.  See the NOTICE file-->
+<!-- distributed with this work for additional information-->
+<!-- regarding copyright ownership.  The ASF licenses this file-->
+<!-- to you under the Apache License, Version 2.0 (the-->
+<!-- "License"); you may not use this file except in compliance-->
+<!-- with the License.  You may obtain a copy of the License at-->
+<!---->
+<!--   http://www.apache.org/licenses/LICENSE-2.0-->
+<!---->
+<!-- Unless required by applicable law or agreed to in writing,-->
+<!-- software distributed under the License is distributed on an-->
+<!-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY-->
+<!-- KIND, either express or implied.  See the License for the-->
+<!-- specific language governing permissions and limitations-->
+<!-- under the License.-->
+<!-- NOTE: For help with the syntax of this file, see:-->
+<!-- http://maven.apache.org/doxia/references/apt-format.html-->
+# Authentication
+
+The most common authentication mechanisms supported by SCM providers are outlined below.
+
+<!-- MACRO{toc|fromDepth=2} -->
+
+## Username/Password Authentication
+
+All SCM providers support a username/password based authentication. As the username and password are transmitted in clear text one should only use this with encrypted transport protocols like HTTPS. 
+
+## SSH Public Key Authentication
+
+Most SCM providers now also support SSH authentication based on an SSH key pair. In order for the server to authenticate a client it must know the client's public SSH. You need to upload the SSH public key to the remote server in order to leverage this authentication and configure your private SSH key for the according repository.
+
+Some information on how to set this up for Git servers can be found at <https://git-scm.com/book/en/v2/Git-on-the-Server-Generating-Your-SSH-Public-Key.html>.
+
+## Configuration
+
+There are multiple ways how the authentication can be configured. They are listed in the recommended order from safest to unsafest option.
+
+### Implicit
+
+Most SCM providers have built-in authentication configuration means. On demand they will interactively ask for credentials (in case Maven is not running in batch mode and the credentials are not yet known) and persist those throughout the session or even longer. Often one can also configure the credentials in advance in dedicated files.
+
+- [Subversion Credentials](https://svnbook.red-bean.com/en/1.8/svn.serverconfig.netmodel.html#svn.serverconfig.netmodel.creds)
+- [Git Credentials](https://git-scm.com/docs/gitcredentials)
+- [Mercurial Credentials](https://www.mercurial-scm.org/help/topics/config#auth)
+
+Providers supporting SSH authentication often leverage the [OpenSSH client configuration file](https://man.openbsd.org/ssh_config).
+
+### Settings.xml
```

Comment:
setting.xml

Reply:
Mine is called `settings.xml` (also correct according to https://maven.apache.org/settings.html).

Reply:
Typo, my nad

Reply:
Agreed